cpp/models/abm/world.h

```diff
@@ -59,8 +59,8 @@ class World
         , m_infection_parameters(params)
         , m_migration_parameters()
         , m_trip_list()
-        , m_use_migration_rules(true)
     {
+        use_migration_rules(true);
```

Comment:
We should maybe make a new issue (if it does not already exist) to implement the possibility to not use these rules. (This shouldnt be done in this issue)

Reply:
see #447